src/shared/components/dateRangePicker/DateRangePicker.tsx

```diff
@@ -35,7 +41,9 @@ class DateRangePicker extends PureComponent<Props, State> {
       timeRange: {lower, upper},
     } = props
 
-    this.state = {lower, upper}
+    // The button can be disabled to start with, only activate it if the values are changed.
```

Comment:
AFAIK the lower & upper are defaulted to a true scenario. It might be worth it for us to calculate the validity of the date range on instantiation rather than assuming it's invalid

Reply:
Like we spoke in our slack conversation, David Kim who's our UX expert was saying its preferable to keep the Button disable until the user makes a change.